content/posts/February-2020.md

```diff
@@ -29,3 +29,36 @@ Kudos to [hhorak] for the help!
 [how to look for RPM spec files on its own]: https://github.com/packit-service/packit/pull/634
 [based on Fedora 31]: https://github.com/packit-service/deployment/pull/51
 [hhorak]: https://github.com/hhorak
+
+## Week 7
+
+The default configuration generated by Packit has [test jobs enabled] from now
+on. This should simplify configuring Packit in new repositories.
+
+[Fixed a bug] which was causing SRPM-build failures in Packit Service for projects
```

Comment:
@lbarcziova is my understanding correct for what packit-service/packit#698 was about?

Reply:
Yes, exactly!

Reply:
Thanks!